.github/workflows/goreleaser-edge.yml

```diff
@@ -0,0 +1,111 @@
+name: goreleaser edge containers
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+env:
+  REGISTRY: docker.io
+
+jobs:
+  goreleaser:
+    permissions:
+      # Add "contents" to write release
+      contents: 'write'
+      # Add "id-token" for google-github-actions/auth
+      id-token: 'write'
+
+    strategy:
+      matrix:
+        os: [linux]
+        arch: ['amd64', '386', 'arm64', 'arm', 'ppc64le']
+
+    runs-on: ubuntu-latest
```

Comment:
I assume we are using self-hosted to speed up the release. Since this workflow will execute 5 builds in parallel it should be okay to use the Github runners. Also in this way we won't block our self-hosted runner. Edge releases are not urgent anyways so even if they are very slow, that should be fine

Reply:
I would go with self-hosted runner. Much faster.

Reply:
I think this will be faster with the Github runners. I assume we have 1 hosted runner right now. This workflow will start 5 jobs in parallel for which it will need 5 runners. That would mean that they will actually be executed sequentially. 

This is just a guess for now since I haven't run this. I would give it a shot with the default runners first and if it's slow, we can easily swap it for our own

Reply:
Given it's a matrix build, I'd agree with using shared runners for now.  We know Go builds can be slower on shared runners, so it's really a question of whether or not the total latency of each job is greater or less than the hit from running them on a single runner.  (There will be additional runners in the near future.)